tgui/packages/tgui/hotkeys.js

```diff
@@ -100,6 +100,34 @@ const getKeyData = (e) => {
   };
 };
 
+const keyCodeToByond = keyCode => {
+  if (keyCode === 16) {return 'Shift';}
+  if (keyCode === 17) {return 'Ctrl';}
+  if (keyCode === 18) {return 'Alt';}
+  if (keyCode === 33) {return 'Northeast';}
+  if (keyCode === 34) {return 'Southeast';}
+  if (keyCode === 35) {return 'Southwest';}
+  if (keyCode === 36) {return 'Northwest';}
```

Comment:
Please for the love of god test if this works for `Home`, `End`, `PageUp`, and `PageDown`.

I am 99% sure pressing`Home` on the keybind entry menu actually inputs `Home` not `Northwest`, so this could possibly mess things up. I cant verify this though, BYOND hub is down.

Reply:
![image](https://user-images.githubusercontent.com/69320440/180652242-2d0d9b63-e47c-4a94-a040-665cc542ef39.png)
Pressing home passes `Northwest` to byond. it appears that nothing listens for them?

Reply:
BYOND turns `Home` into `Northwest` and I hate it.

Reply:
Also can you convert these these into a dictionary, like `{33 : 'Northeast', 34 : 'Southeast'}` etc. instead of the if else chain? You will probably still need ifs for the ones where you're working out logic though.

Reply:
done